packages/asgardex-bitcoin/__tests__/client.test.ts

```diff
@@ -159,17 +159,17 @@ describe('BitcoinClient Test', () => {
     btcClient.setNetwork('testnet')
     btcClient.setPhrase(phraseOne)
     const estimates = await btcClient.getFees()
-    expect(estimates.fast).toEqual(expect.any(Number))
-    expect(estimates.fastest).toEqual(expect.any(Number))
-    expect(estimates.average).toEqual(expect.any(Number))
+    expect(estimates.fast).not.toBeNull()
+    expect(estimates.fastest).not.toBeNull()
+    expect(estimates.average).not.toBeNull()
```

Comment:
Since `estimates.fast` (etc.) can be `undefined` (but not `null`) use `toBeDefined`. 
Test against `not.toBeNull()` won't work in case of `undefined`.
`

Reply:
Fixed